README.md

```diff
@@ -1,17 +1,25 @@
-# GAN Metrics
+GAN Metrics
+------
 
 This repository provides the code for [An empirical study on evaluation metrics of generative adversarial networks](https://arxiv.org/abs/1806.07755).
 
-## Requirement
+Requirement
+------
+
 - Python 3.6.4
 - torch 0.4.0
 - torchvision 0.2.1
-- ot 0.4.0
+- pot 0.4.0

```

Comment:
Why do you change it to `pot`?

Reply:
Because the project official name is [POT](https://pypi.org/project/POT/) although we `import ot`. It takes me a while to find out that `ot` means `POT`, and then we can use `pip install pot` to install it.

Reply:
Ah, I didn't realize this. Thanks for pointing out! Can you also fix the readme in the framework folder accordingly?